bin/setup

```diff
@@ -0,0 +1,63 @@
+                    `:/ .  +`-:-    +--/.:                                                          
+                 :.../` `  .+dd-    :yh. /                                                          
+                 :`//d`.`    ` `s+/-     o                 -`                                       
+                 --:oyo :       /::-  ` //            ` `-/+/-.-/s+                                 
+                  --/-::   -`         ./-            +y: :.   .-.  `-/o-                            
+                  .`   `:`   .  `-   `:/`          .::/:     :-:.   -:.-`                           
+                        .-.-.:`  ..:/-            sodd+-.   -dy`/.  .`  :y.`:/.                     
+                             `-:-`                :`:+/:`    -:-:  ./-   :+o/`+                     
+                                                 .o     -:--//     ++-:`+`ho+ o                     
+                                                 .y `.  +-shs-     ./-:`  h++ +                     
+                                                  //`.   :/:.     --     -h:--s                     
+                                                  :y/ `.     -:   ``  .+.:.                         
+                                                    -+..         `     /y                           
+                                                     :ho:`  :`   :` ./.                             
+           .++///:::::::::::::::                        `/yo/--/++shy:                              
+         `hs`          .+-                                ``   .+`                                  
+         yh          `ss`                                          `-                               
+         om`        :m:`-                                  `::`    ++    ``              /`         
+          :o+//::: om-:ho`.s`   :d` `/so+/+o   -oo/:++-  /s:`:h  ::  `o++:o.  /+    so  +d          
+       +yso++-  `/dd:oy  oh`   od` /d/   -h` `hs`  `y/  hs   .``y+  +d. s`+..d+   .d+ `o`y/         
+      sh     .` .ms :s  od`  -dd``yN`   :d` /N+   -d/ .yN     -mo `yN`  `ss:N+   +N+ :+  do`+`      
+      -h.     .sh-      ss.:+:N://:h``-sm-//-N:`-//m:/:`d/.-//:m////d``:+- `N-./+oh:++..sN+:        
+        ::::/++.         .-`  ``   `::/m+-   `--`        `.`        `--`     -.   ``  -::`          
+                                  `/+oN:                                                            
+                                .so`.m:                                                             
+                               /y` -d-                                                              
+                               h..oo`                                                               
+                                -.                                                                  
+
+  ]
+
+
+  puts "\n== Installing dependencies =="
+  run "gem install bundler --conservative"
```

Comment:
--no-ri --no-rdoc may be worth adding to speed things up, although those things perhaps are best left to each dev via their `.gemrc` config.